src/script/page/RightSidebar/ConversationDetails/ConversationDetails.tsx

```diff
@@ -89,6 +90,8 @@ const ConversationDetails: FC<ConversationDetailsProps> = ({
   isFederated = false,
   isVisible = false,
 }) => {
+  const panelContentRef = useRef<HTMLDivElement>(null);
```

Comment:
you will need a `useEffectRef` here, cause we need a rerender when the ref is updated (ref do not trigger a re-render on their own)

Reply:
note for the future:
- let's migrate all `useFadingScrollbar` calls to an simple function that is passed to the `ref` attributes of elements. 

A `ref` + `hook` is not needed and not ideal